homemade.html

```diff
@@ -0,0 +1,308 @@
+            <h1 class="display-5 p-3 text-decoration-underline">Creamy chicken pesto pasta</h1>
+
+            <div class="row text-center g-4 d-md-flex">
+                <div class="col-xl-3 p-5">
+                    <h1 class="display-6 fw-bold">Ingredients (9)</h1>
+                    <p class="card-text text-dark mt-2 fs-4 ">
+                        
+                            -350g Cucina Matese Penne Rigate
+                            <br>-300ml light cream
+                            <br>-1/2 cup basil pesto
+                            <br>-3 green onions, sliced
+                            <br>-1 1/2 cups skinless shredded roast chicken (see related recipe)
+                            <br>-1/2 cup drained sun-dried tomatoes, thinly sliced
+                            <br>-1/3 cup finely grated parmesan cheese
+                            <br>-Thinly sliced green onions, to serve
+                            <br>-Crusty bread, to serve</p>
+                </div>
+                <div class="col-xl-3 p-5">
+                    
+                    <h1 class="display-6 fw-bold">Steps</h1>
+                    <p class="card-text text-dark mt-2 fs-4">
+                        <br><span class="fw-bold fs-4">Step 1-</span>
+                        Cook 350g Cucina Matese Penne Rigate in a saucepan of boiling, salted water, following packet directions, until tender. Drain, reserving 1/4 cup liquid.
+            
+                        <br><span class="fw-bold fs-4">Step 2-</span>
+                        Return pasta to pan over medium-low heat. Add reserved liquid, 300ml light cream, 1/2 cup basil pesto, 3 green onions, sliced, 1 1/2 cups skinless shredded roast chicken (see related recipe) and 1/2 cup drained sun-dried tomatoes, thinly sliced. Stir to combine. Cook, stirring, for 1 to 2 minutes or until heated through.
+
+                        <br><span class="fw-bold fs-4">Step 3-</span>
+                        Divide between bowls. Top with 1/3 cup finely grated parmesan cheese and Thinly sliced green onions, to serve. Serve with Crusty bread, to serve.                       </p>
+                </div>
+                <div class="col-xl-6 p-2">
+                    <img class="img-fluid w-100 h-95 m-3" src="/img/dish1.jpg" alt="">
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="p-5">
+        <div class="container-fluid aling-items-centar">
+            <h1 class="display-5 p-3 text-decoration-underline">Chicken and mushroom risotto</h1>
+            <div class="row text-center g-4 d-md-flex">
+                <div class="col-xl-2 p-5">
+                    <h1 class="display-6 fw-bold">Ingredients(12)</h1>
+                    <p class="card-text text-dark mt-2 fs-4 fst">
+                        
+                        
+                        -2.5L (10 cups) Massel chicken style liquid stock
+                    <br>-40g Coles Australian Salted Butter

```

Comment:
izbjegavamo korištenje <br> tagova (samo neki specijalni slučaji kao što je tekst u nekom naslovu), ovo se može sa <span> ILI <ul> ILI svaki element s <p>